spec/fixtures/project_dir/bolt.yaml

```diff
@@ -0,0 +1,2 @@
+modulepath: modules:../modules
+project_name: true
```

Comment:
I assumed that the value of `project_name` would be the name to use for the project.

Reply:
Yeah I misunderstood the original issue - I'm reworking the Puppet side now and will update shortly.